### PR TITLE
Added binary function wrappers

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
My first PR for intercept, would be great if I could get some feedback if I'm doing something wrong here.

### CODE REVIEW

File: src/client/intercept/client/sqf/uncategorized.cpp [hunk 1]
```diff
@@ -286,6 +286,42 @@ namespace intercept {
...
 		}
 
+        void add_backpack_cargo(const object & vehicle_, const std::string & packClassName_, int count_) {
+            std::vector<game_value> args{
+                packClassName_,
+                (float)count_
+            };
+
+            host::functions.invoke_raw_binary(client::__sqf::binary__addbackpackcargo__object__array__ret__nothing, vehicle_, args);
+		}
```
Comment (Glowbal) [inline src/client/intercept/client/sqf/uncategorized.cpp]: Something wierd is going on here with your spacing

File: src/client/intercept/client/sqf/uncategorized.hpp [hunk 2]
```diff
@@ -190,6 +196,9 @@ namespace intercept {
 
 		/* potential namespace: misc */
 		void create_guarded_point(const side &side_, const vector3 &pos_, float idstatic_, const object &veh_);
+		void action(const object &unit_, const std::vector<game_value> &action_array_);
+        float add_event_handler(const object &object_, const std::string &type_, const game_data_code &command_);
```
Comment (esteldunedain) [inline src/client/intercept/client/sqf/uncategorized.hpp]: inconsistent indentation

File: src/client/intercept/client/sqf/uncategorized.cpp [hunk 3]
```diff
@@ -1931,6 +2036,57 @@ namespace intercept {
...
+        void add_curator_addons(const object & curator_object_, const std::vector<std::string>& addons_) {  
+            std::vector<game_value> addons;
+            for (std::string addon : addons_) {
+                addons.push_back(game_value(addon));
+            }
+            host::functions.invoke_raw_binary(client::__sqf::binary__addcuratoraddons__object__array__ret__nothing, curator_object_, addons);
+        }
+
+        void add_curator_camera_area(const object & curator_object_, int camera_area_id_, const vector2 & position_, float radius_) {
+            std::vector<game_value> args{
```
Comment (Glowbal) [inline src/client/intercept/client/sqf/uncategorized.cpp]: This has to be 
```c++
game_value args(std::vector<game_value>{
        (float)camera_area_id_,
        position_,
        radius_
});
```

File: src/client/intercept/client/sqf/uncategorized.cpp (hunk 1 above)
Comment (BaerMitUmlaut) [inline src/client/intercept/client/sqf/uncategorized.cpp]: Fixed (there were two tabs there).

File: src/client/intercept/client/sqf/uncategorized.hpp (hunk 2 above)
Comment (BaerMitUmlaut) [inline src/client/intercept/client/sqf/uncategorized.hpp]: Fixed.

**Comment by NouberNou** [comment]
Looks good to me.

**Comment by esteldunedain** [comment]
@BaerMitUmlaut I still see most of the lines wrongly indented

**Comment by BaerMitUmlaut** [comment]
@esteldunedain: The majority of the file uses tabs for identation. According to the [coding standard](https://github.com/NouberNou/intercept/wiki/Coding-Standards) four spaces should be used though, which I am doing. Same thing with the curly brackets.

**Comment by esteldunedain** [comment]
> @esteldunedain: The majority of the file uses tabs for identation. According to the coding standard four spaces should be used though, which I am doing. Same thing with the curly brackets.

Aha, ok. Good boy then

**Comment by Glowbal** [comment]
Thanks @BaerMitUmlaut

**Pull request merged**